ppsim/simulator.pyx

```diff
@@ -392,13 +392,13 @@ cdef class SimulatorMultiBatch(Simulator):
         else:
             return self.delta[a, b, coin], self.delta[a, b, 1-coin]
 
-    # TODO: fix
     def get_enabled_reactions(self):
         """Updates :any:`enabled_reactions` and :any:`num_enabled_reactions`."""
-        cdef npy_intp i
+        cdef npy_intp i, j, k
         self.num_enabled_reactions = 0
         for i in range(len(self.reactions)):
-            if self.config[self.reactions[i][0]] > 0 and self.config[self.reactions[i][1]] > 0:
+            j, k = self.reactions[i][0], self.reactions[i][1]
```

Comment:
I'd recommend more descriptive names for these variables than `j` and `k`.

Reply:
This was to save space over inlining 'self.reactions[i][0]' and 'self.reactions[i][1]', which appear 7 times in the next line.
Writing something like reactant_1, reactant_2 would sort of defeat that purpose.

Reply:
I personally find it easier to read and understand the code with descriptive variable names (such as `reactant_1`) even if it is a bit longer. (and it's also easier to read than using `self.reactions[i][0]`)